src/main/java/com/uid2/admin/vertx/AdminVerticle.java

```diff
@@ -147,6 +161,11 @@ private void handleTokenGet(RoutingContext rc) {
         });
     }
 
+    private void respondWithTestAdminUser(RoutingContext rc) {
+        // This test user is set up in src/main/resources/localstack/s3/admins/admins.json
+        handleEmailContactInfo("test.user@uidapi.com", rc);

```

Comment:
could we default and make this configurable in local-config.json as AdminVerticle.config already reads from it?

Reply:
I could move it there but as described in the config, the email needs to match the one in mock data. In a config file there would be no way to add this comment.